daemon/execdriver/driver.go

```diff
@@ -125,18 +125,20 @@ type UTS struct {
 // Currently these are all for cgroup configs.
 // TODO Windows: Factor out ulimit.Rlimit
 type Resources struct {
-	Memory           int64            `json:"memory"`
-	MemorySwap       int64            `json:"memory_swap"`
-	KernelMemory     int64            `json:"kernel_memory"`
-	CPUShares        int64            `json:"cpu_shares"`
-	CpusetCpus       string           `json:"cpuset_cpus"`
-	CpusetMems       string           `json:"cpuset_mems"`
-	CPUPeriod        int64            `json:"cpu_period"`
-	CPUQuota         int64            `json:"cpu_quota"`
-	BlkioWeight      int64            `json:"blkio_weight"`
-	Rlimits          []*ulimit.Rlimit `json:"rlimits"`
-	OomKillDisable   bool             `json:"oom_kill_disable"`
-	MemorySwappiness int64            `json:"memory_swappiness"`
+	Memory                      int64            `json:"memory"`
+	MemorySwap                  int64            `json:"memory_swap"`
+	KernelMemory                int64            `json:"kernel_memory"`
+	CPUShares                   int64            `json:"cpu_shares"`
+	CpusetCpus                  string           `json:"cpuset_cpus"`
+	CpusetMems                  string           `json:"cpuset_mems"`
+	CPUPeriod                   int64            `json:"cpu_period"`
+	CPUQuota                    int64            `json:"cpu_quota"`
+	BlkioWeight                 int64            `json:"blkio_weight"`
+	BlkioThrottleReadBpsDevice  string           `json:"blkio_throttle_read_bps_device"`
```

Comment:
Should be a strong type with serialization to whatever format libcontainer expects (which would prefferably be a strong type as well, but I'm sure that's probably already in).